docs/integrations/sources/shopify.md

```diff
@@ -100,6 +100,7 @@ This is expected when the connector hits the 429 - Rate Limit Exceeded HTTP Erro
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.1.35 | 2022-03-07 | [10915](https://github.com/airbytehq/airbyte/pull/10915) | Implemented independent state for each ShopifySubStream |
```

Comment:
framing it in a way a non-engineer can understand ;) 

```suggestion
| 0.1.35 | 2022-03-07 | [10915](https://github.com/airbytehq/airbyte/pull/10915) | Fix a bug which caused full refresh syncs of child REST entities configured for incremental |
```

Reply:
Acceptable. Thank you